cpp/oneapi/dal/data/array_dpc_test.cpp

```diff
@@ -108,17 +108,36 @@ TEST(array_dpc_test, can_reset_array_with_raw_pointer) {
 
     constexpr int64_t count = 10;
     auto* data              = sycl::malloc_shared<float>(count, q);
-    auto event              = q.submit([&](sycl::handler& cgh) {
-        cgh.parallel_for(sycl::range<1>(count), [=](sycl::id<1> idx) {
-            data[idx[0]] = idx;
-        });
-    });
+    q.submit([&](sycl::handler& cgh) {
+         cgh.parallel_for(sycl::range<1>(count), [=](sycl::id<1> idx) {
+             data[idx[0]] = idx;
```

Comment:
Do you mean 
data[idx[0]] = static_cast<float>(idx[0]);

Reply:
yes, thanks